docs/UpstreamManagement.rst

```diff
@@ -23,3 +23,7 @@ In that case the upstream is not removed.
 .. figure:: pictures/upstream_scheme.png
 
    Installation scheme of upstream instances. If tag v0.18.0.5 is the oldest tag supported, the orange upstream instances can safely be removed.
+
+Upgrade sysconfigs
+----------------------
+To copy the alp's env files, use https://github.com/C2SM/spack-c2sm/blob/main/sysconfigs/copy_alps_env_files_from.sh
```

Comment:
would need a bit more - to me the is related to a new upstream , for example:
When CSCS provide an update software stack upstream on alps the corresponding environement file need to be updated. The scripts ./sysconfigs/copy_alps_env_files_from.sh  can be used

SHoudl you not call it from inside the spack-c2sm , otherwise how do you copy at the write place (since you provide a http link is not clear)

Reply:
Addressed in next commit.